packages/lesswrong/server/rateLimitUtils.ts

```diff
@@ -185,10 +196,12 @@ export async function rateLimitDateWhenUserNextAbleToComment(user: DbUser, postI
   if (ignoreRateLimits) return null;
 
   // does the user have a moderator-assigned rate limit?
-  const [modRateLimitHours, modPostSpecificRateLimitHours, userCommentRateLimit] = await Promise.all([
+  // also get the recent karma info, we'll need it later
+  const [modRateLimitHours, modPostSpecificRateLimitHours, userCommentRateLimit, recentKarmaInfo] = await Promise.all([
     getModRateLimitHours(user._id),
     getModPostSpecificRateLimitHours(user._id),
-    getUserRateLimit(user._id, 'allComments')
+    getUserRateLimit(user._id, 'allComments'),
+    getRecentKarmaInfo(user._id)
```

Comment:
Just some performance improvement by getting the recentKarmaInfo at the same time as everything else